src/atoms/Photography/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from "react"
+import styles from "./photography.module.css"
+
+class Photography extends Component {
```

Comment:
We could find a better (not so generic) name for this Component... and also turn it into a Functional Component.

Reply:
I gave it the name HeroImage, what do you think?